compass/ocean/tests/baroclinic_channel/baroclinic_channel.cfg

```diff
@@ -48,4 +51,4 @@ gradient_width_dist = 40e3
 salinity = 35.0
 
 # Coriolis parameter for entire domain.
-coriolis_parameter = -1.2e-4
\ No newline at end of file
+coriolis_parameter = -1.2e-4
```

Comment:
@xylar I haven't seen this before. I'm not sure how to fix it. Does it mean there should be a line at the end of the document?

Reply:
The lack of a new line before your changes is undesirable so what you've done here is correct.  Thank you!